templater/jobs/presubmit/eks-anywhere-build-tooling/aws-image-builder-presubmits.yaml

```diff
@@ -0,0 +1,13 @@
+jobName: aws-image-builder-presubmit
+runIfChanged: ^build/lib/.*|Common.mk|projects/aws/image-builder/.*
+imageBuild: true
+commands:
+- if $(make check-project-path-exists); then make build -C $PROJECT_PATH; fi
```

Comment:
@jaxesn I agree that it will be best to add this in a template as most of the jobs have this check. Already mentioned this to Abhay but thought it will be better to focus on finishing the minimal configs first.